extensions-builtin/Lora/ui_extra_networks_lora.py

```diff
@@ -16,7 +16,9 @@ def refresh(self):
         networks.list_available_networks()
 
     def create_item(self, name, index=None, enable_filter=True):
-        lora_on_disk = networks.available_networks.get(name)
+        lora_on_disk = networks.available_networks.get(name, None)

```

Comment:
somehow previously I managed to trigger an error when using get(key)
not entirely sure how
that's why I explicitly specified None forgetting it doesn't make a difference
strangely after I switch to None the error didn't appear
so ya if it can be simplified why not